Solutions/Branwen/GUI.cs

```diff
@@ -0,0 +1,166 @@
+﻿using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;

```

Comment:
please consider adding a packages.config file for this in the future, nuget will automatically get it for the user in the future